www/src/components/publisher/PublisherSideNav.tsx

```diff
@@ -12,6 +12,7 @@ function PublisherSideNav({ publisher }: any) {
     >
       <PageCard
         icon={{
+          name: capitalize(publisher.name),
```

Comment:
are we sure we should capitalize that?  I feel like just putting the raw string is correct.

Reply:
I think you are right. `PageCard` takes the name and creates initials. The upper case transformation is probably already handled there.